src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const trs = document.querySelectorAll('tr');
+
+trs.forEach((el) => {
+  const parentElement = el;
+
+  if (parentElement.children.length > 1) {
+    const firstChild = parentElement.children[1];
```

Comment:
The task requires cloning the second column of the table, but the code is currently selecting the second child of each row. Ensure that the second column is being targeted correctly, considering the table structure with `thead`, `tbody`, and `tfoot`.